src/media/findWidget.ts

```diff
@@ -66,7 +140,11 @@ export function initializeFindWidget(document: Document, onAddNode: (formDataJSO
 		// copy properties from form data to json object
 		const formDataJSON = {} as any;
 		formData.forEach((value, key) => formDataJSON[key] = value);
-		onAddNode(new FindWidgetFormData(formDataJSON));
+
+		findWidgetFormState.update(formDataJSON);
```

Comment:
It is a bad practice to put everything into gui functions, this should probably be extracted as some method of the state class.